README.md

```diff
@@ -1,14 +1,20 @@
 
-Halcyon is different and incompatible with both Second Life© and OpenSims, although we can use the OpenSim viewers (<a href="https://firestormviewer.org/os-operating-system/" rel="follow" >FirestormOS</a> recommended). Halcyon has better physics & script engines, more prims, free upload (depending on each world's setting), and a more powerful asset database (NOT a "cloud" ... <a href="https://fsfe.org/contribute/spreadtheword.html#nocloud" rel="follow" >*there is no "cloud" only somebody else's computers*</a>). As a proof, this <a href="https://youtube.com/watch?v=_QnNH-xDPyg" rel="nofollow">tram</a> has no equivalent in any of the SL or OS worlds. Halcyon is intended for anybody to run their own world, and there already are several. With projects of inter-worlds communication. Although this needs a restart of the Halcyon development, which has been very slow since the end of <a href="https://web.archive.org/web/20180806205343/https://inworldz.com/" rel="nofollow" >InWorldz</a>. A project which is today coming to fruition is <a href="https://mundosmarket.com" rel="follow" >Mundos Market</a>, which will allow merchants to sell in all connected Halcyon Worlds.
+# Different 
+
+Halcyon is different and incompatible with both Second Life™ and OpenSims, although we can use the OpenSim viewers (<a href="https://firestormviewer.org/os-operating-system/" rel="follow" >FirestormOS</a> recommended, and currently <a href="https://discover.creomundos.com/ViewerDownloads.aspx" rel="follow">6.6.14</a> is the last compatible with HalcyonGrid). Halcyon has better physics & script engines, more prims, free upload (depending on each individual world's setting), and a more powerful asset database (Not a "Cloud" ... <a href="https://fsfe.org/contribute/spreadtheword.html#nocloud" rel="follow" >*there is no "cloud" just other people's computers*</a>). As a proof, <a href="https://youtube.com/watch?v=_QnNH-xDPyg" rel="nofollow">this tram</a> has no equivalent in any of the SL or OS worlds. 
```

Comment:
I think this entire paragraph needs a rethink from the ground up. There's no such thing as `OpenSims`, there is "OpenSim" and there are other forks thereof.I recommend moving the FirestormOS recommendation out of the parenthetical block. In fact I'm not quite sold that this paragraph nor this readme is the best place to be recommending viewers anyway.

Jabs and snide comments don't belong in README files.